common/scripts/create-rshiny.sh

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env bash
```

Comment:
@gerardcl Could you move that script out of `common`? I would put this inside the rshiny folder.

Reply:
@michaelsauter shall I move out the Tailor-related files from common too?

Reply:
`common` should only contain things that at least two quickstarters use I'd say. So if there are files that are ONLY used by rshiny, the files should be in rshiny.

Reply:
then we keep that in common, since that is used for jupyter too, but will rename it

Reply:
Haha. Cool. My recent PR #49 introduces some merge conflicts now - should be easy to resolve though ...